### PR TITLE
added check_yum_exclude_list check

### PR DESCRIPTION
**Pull request by jameerpathan111**
Fixes issue #66 
Test Results:
```
$ pytest --ansible-host-pattern satellite --ansible-user=root  --ansible-inventory testfm/inventory tests/test_health.py -k test_positive_check_yum_exclude_list -svvv
=================================================================================== test session starts ===================================================================================
collected 15 items / 13 deselected                                                                                                                                                        

tests/test_health.py::test_positive_check_yum_exclude_list 2019-05-31 12:11:17,198 - testfm.log - INFO - Running ForemanMaintain::Scenario::FilteredScenario
================================================================================
Check if yum exclude list is configured:                              [FAIL]
The /etc/yum.conf has exclude list configured as below,
  exclude=cat* bear*
Unset this as it can cause yum update or upgrade failures !
--------------------------------------------------------------------------------
Scenario [ForemanMaintain::Scenario::FilteredScenario] failed.

The following steps ended up in failing state:

  [check-yum-exclude-list]

Resolve the failed steps and rerun
the command. In case the failures are false positives,
use --whitelist="check-yum-exclude-list"
PASSED
tests/test_health.py::test_positive_check_yum_exclude_list_without_excludes 2019-05-31 12:11:34,905 - testfm.log - INFO - Running ForemanMaintain::Scenario::FilteredScenario
================================================================================
Check if yum exclude list is configured:                              [OK]
--------------------------------------------------------------------------------
PASSED

======================================================================== 2 passed, 13 deselected in 42.41 seconds =========================================================================
```

### CODE REVIEW

**Comment by ntkathole** [comment]
[test]

File: tests/conftest.py [hunk 1]
```diff
@@ -14,6 +14,26 @@
 from testfm.log import logger
 
 
+@pytest.fixture(scope='function')
+def setup_yum_exclude(request, ansible_module):
+    """This fixture is used for adding and then removing yum excludes in /etc/yum.conf file.
+    """
+    exclude = 'exclude=cat* bear*'
```
Comment (ntkathole) [inline tests/conftest.py]: It will be good to make such fixtures more generic. Can we pass this "exclude = 'exclude=cat* bear*'" as a variable to fixture?

Comment (jameerpathan111) [reply tests/conftest.py]: will check if we can do it.

**Comment by ntkathole** [comment]
Will merge when check will be in downstream.

**Comment by jameerpathan111** [comment]
@ntkathole can we merge this?

**Pull request merged**